rust/main/hyperlane-base/src/cache/moka/hyperlane_cache.rs

```diff
@@ -0,0 +1,95 @@
+use super::{BaseCache, CacheResult, Expiration, ExpirationType};
+use crate::cache::{CacheError, HyperlaneCache};
+use async_trait::async_trait;
+use hyperlane_core::H256;
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
+use std::fmt::Debug;
+
+/// Cache for storing function calls with serializable results
+#[derive(Debug, Clone)]
+pub struct HyperlaneMokaCache(BaseCache);
```

Comment:
If we rename `HyperlaneCache` into (for example) `ContractMethodCallCache`, would it make sense to hide the implementation detail about this cache (specifically `Moka` part) and call it `LocalContractMethodCallCache` (as example)?

Reply:
As discussed, renamed to `LocalCache` to be more functionality specific 

[Fix](https://github.com/Mantas-M/hyperlane-monorepo/commit/fcdb62c2f7fcd22417178905a6341303e9d2fd7f)